src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.hpp

```diff
@@ -266,6 +266,8 @@ class ShenandoahHeapRegion {
 
   ShenandoahSharedFlag _recycling; // Used to indicate that the region is being recycled; see try_recycle*().
 
+  bool _need_bitmap_reset;
```

Comment:
Nit pick, but I think this would read better as `_needs_bitmap_reset`.

Reply:
Done.